_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -602,15 +602,17 @@ public static function validate_setup_wizard_questionnaire( $value, $request, $p
 	}
 
 	public static function get_plans( $request ) {
-		$request = Client::wpcom_json_api_request_as_user(
+		$request = Client::wpcom_json_api_request_as_blog(
```

Comment:
This is also something we may want to fix regardless. I don't know why we need a specific user to request the plans